airflow/www/templates/airflow/tree.html

```diff
@@ -1,13 +1,13 @@
-{# 
+{#
   Licensed to the Apache Software Foundation (ASF) under one or more
   contributor license agreements.  See the NOTICE file distributed with
   this work for additional information regarding copyright ownership.
   The ASF licenses this file to You under the Apache License, Version 2.0
   (the "License"); you may not use this file except in compliance with
   the License.  You may obtain a copy of the License at
-  
+
```

Comment:
@aoen Hmm maybe my editor (Atom) removed the white space automatically but the Apache license content should be exactly the same.